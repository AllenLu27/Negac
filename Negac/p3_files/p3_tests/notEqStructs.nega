struct A {
	bool field;
};

bool v(){
	struct A first;
	struct A second;
	return first != second;
}

