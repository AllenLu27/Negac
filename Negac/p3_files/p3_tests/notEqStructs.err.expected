FATAL [8,15]: Invalid equality operation
Semantic analysis failed
